tests/test_inheritance_iterable.py

```diff
@@ -8,7 +8,6 @@
 
 
 class InheritanceIterableTest(TestCase):
-    @skipIf(django.VERSION[:2] == (1, 10), "Django 1.10 expects ModelIterable not a subclass of it")
```

Comment:
No longer need to import `django` or `skipIf` into this file? (Why didnt't flake8 fail? :thinking: )

Reply:
Good point. the project doesn't use `flake8`. I have a [PR to introduce black](https://github.com/jazzband/django-model-utils/pull/454).
I will remove `django` and `skipIf`